### ISSUE TITLE
Unable to download mac version

### ISSUE THREAD

**Issue by javalnanda** [codefoxes/firebase-admin#37]
There is some issue with the download link of Mac version. Every time it fails to download after 15-18 MB of progress.
Anyone else facing this issue? I tried almost 6-8 times and every time it fails after 15 MB of data is downloaded.

**Comment by umarhussain15** [comment]
yes the server failing for the download of windows version as well 
![image](https://user-images.githubusercontent.com/14268893/28658283-f5495ea2-72c3-11e7-953f-29aaf3495a87.png)

**Comment by karthikax** [comment]
That looks like cloudflare issue. Can you try to download now?

**Comment by vbGoz** [comment]
I am still not able to download.

**Comment by steven-fernandez** [comment]
Same problem here
